app/web/.env.development

```diff
@@ -1,6 +1,6 @@
 NEXT_PUBLIC_COUCHERS_ENV=dev
-NEXT_PUBLIC_API_BASE_URL=http://localhost:8888
-NEXT_PUBLIC_MEDIA_BASE_URL="http://localhost:5000"
```

Comment:
I think we should keep these so those of us who want to run everything locally don't have to update this separately, which seems suiting for dev environment variables

Reply:
I agree, please see below.
The default when running `docker-compose up` will now be to use these localhost URLs

Reply:
Was missing a file here to allow local dev to work (.env.local). Now tested working.